boards/scdz/ch32v307evt/doc/index.rst

```diff
@@ -0,0 +1,91 @@
+.. zephyr:board:: ch32v307evt
```

Comment:
Do you have a link to the board (home page, store link, or similar) and schematic?